Plugins/PauseMyScreen/Plugin.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Text.RegularExpressions;
+
+using TrainingServer;
+using TrainingServer.Extensibility;
+
+namespace FreezeScene;
+public class Plugin : IPlugin
+{
+#if DEBUG
+	public string FriendlyName => "Pause my screen (DEBUG)";
+#else
+	public string FriendlyName => "Pause my screen";
+#endif
+	public string Maintainer => "Álex (605126)";
+
+	private readonly Regex _pause, _resume;
+
+	public Plugin()
+	{
+
+		string[] regexes = new[]
+		{
+			@"PAUSE$",
+			@"RESUME$"
+		};
+
+		if (File.Exists("PauseMyScreenCommands.re") && File.ReadAllLines("PauseMyScreenCommands.re").Length >= 1)
+			regexes = File.ReadAllLines("PauseMyScreenCommands.re");
+		else
+			File.WriteAllLines("PauseMyScreenCommands.re", regexes);
+
+		_pause = new(regexes[0], RegexOptions.IgnoreCase);
+		_resume= new(regexes[1], RegexOptions.IgnoreCase);
```

Comment:
```suggestion
		_pause = new(regexes[0], RegexOptions.IgnoreCase | RegexOptions.SingleLine | RegexOptions.Compiled);
		_resume= new(regexes[1], RegexOptions.IgnoreCase | RegexOptions.SingleLine | RegexOptions.Compiled);
```

Consider `RegexOptions.SingleLine` in case someone puts some strange characters in a message.